src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_CompoundAssignmentOperator.cs

```diff
@@ -730,6 +730,11 @@ private BoundExpression BoxReceiver(BoundExpression rewrittenReceiver, NamedType
                 case BoundKind.Local:
                     return localsMayBeAssignedOrCaptured || ((BoundLocal)expression).LocalSymbol.RefKind != RefKind.None;
 
+                case BoundKind.TypeExpression:
+                    return false;
+
+                // CONSIDER: Could add support for a (converted) method group.  It cannot change if the receiver cannot change.
```

Comment:
Please handle or file a bug.

Reply:
[https://github.com/dotnet/roslyn/issues/37120](https://github.com/dotnet/roslyn/issues/37120)

---
In reply to: [301354753](https://github.com/dotnet/roslyn/pull/37052#discussion_r301354753) [](ancestors = 301354753)